layouts/shortcodes/pipelines/latest-version.html

```diff
@@ -1 +1 @@
-1.8.5
\ No newline at end of file
+1.8.19
```

Comment:
Please note that this 1.8.19 is SDK version, not the backend version. Latest official Backend version is https://github.com/kubeflow/pipelines/releases/tag/1.8.5